docs/concepts.md

```diff
@@ -2,20 +2,30 @@
 
 Fleet is fundamentally a set of Kubernetes custom resource definitions (CRDs) and controllers
 to manage GitOps for a single Kubernetes cluster or a large scale deployment of Kubernetes clusters
-(up to one million). Below are some of the concepts of Fleet that will be useful through out this documentation.
+(up to one million). 
+
+Below are some of the concepts of Fleet that will be useful throughout this documentation:
 
 * **Fleet Manager**: The centralized component that orchestrates the deployments of Kubernetes assets
-    from git. In a multi-cluster setup this will typically be a dedicated Kubernetes cluster. In a
-    single cluster setup the Fleet manager will be running on the same cluster you are managing with GitOps.
-* **Fleet controller**: The controller(s) running on the Fleet manager orchestrating GitOps. In practice
-    Fleet manager and Fleet controllers is used fairly interchangeably.
+    from git. In a multi-cluster setup, this will typically be a dedicated Kubernetes cluster. In a
+    single cluster setup, the Fleet manager will be running on the same cluster you are managing with GitOps.
+* **Fleet controller**: The controller(s) running on the Fleet manager orchestrating GitOps. In practice,
+    the Fleet manager and Fleet controllers are used fairly interchangeably.
 * **Single Cluster Style**: This is a style of installing Fleet in which the manager and downstream cluster are the
     same cluster.  This is a very simple pattern to quickly get up and running with GitOps.
 * **Multi Cluster Style**: This is a style of running Fleet in which you have a central manager that manages a large
     number of downstream clusters.
 * **Fleet agent**: Every managed downstream cluster will run an agent that communicates back to the Fleet manager.
     This agent is just another set of Kubernetes controllers running in the downstream cluster.
 * **GitRepo**: Git repositories that are watched by Fleet are represented by the type `GitRepo`.
+
+>**Example installation order via `GitRepo` custom resources when using Fleet for the configuration management behind clusters:**
+>
+> 1. Install Calico CRDs and controllers.
```

Comment:
We should link Calico: https://github.com/projectcalico/calico

Reply:
Linked as suggested.